### PR TITLE
desktop: Add URL decode to desktop window's title (close: #15454)

### PR DESCRIPTION
**Pull request by TheDecimeter**
Fix so that desktop title properly renders non ASCII characters (the web player uses a static title so no fix is needed there)

issue: https://github.com/ruffle-rs/ruffle/issues/15454

image

### CODE REVIEW

File: desktop/src/player.rs [hunk 1]
```diff
@@ -184,7 +185,9 @@ impl ActivePlayer {
             .unwrap_or_else(|| movie_url.as_str())
             .to_string();
 
-        window.set_title(&format!("Ruffle - {name}"));
+        let readable_name = decode(&name).expect("UTF-8");
```
Comment (Lord-McSweeney) [inline desktop/src/player.rs:188]: Hmm, this probably shouldn't panic (a SWF file with an invalid name should still play). Can this be something like `unwrap_or(name)`?

**Review by Dinnerbone — APPROVED** [review]

**Comment by Dinnerbone** [comment]
Thank you!

**Comment by torokati44** [comment]
Thank you indeed, and welcome! :wave:

**Comment by TheDecimeter** [comment]
> @TheDecimeter,
> 
> many thanks for your implementation.
> 
> Now animation filenames such as this are possible 😁:
> 
> ruffle-non-ascii-fs8-fs8

My pleasure, it was a fun intro to Rust

**Pull request merged**